### PR TITLE
📖 Rephrased the description after the Documentation header

### PR DESCRIPTION
**Pull request by thealiilman**
### Changes
- Rephrased the description after the Documentation header.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,7 +94,7 @@ You can read more about **new Date()** [here](https://developer.mozilla.org/en-U
 > https://discord.gg/EvbMshU
 
 ### Sources: 
-> https://www.worldometers.info/coronavirus/ 
+> https://www.worldometers.info/coronavirus/
```
Comment (lovung) [inline README.md]: Need to add 1 more line here to separate 2 lines of quotes

Comment (buster95) [reply README.md]: was added in #193

Comment (thealiilman) [reply README.md]: @buster95 I've removed this change. ✌️

**Comment by thealiilman** [comment]
Thanks @ebwinters! 🙂

**Pull request merged**